README.md

```diff
@@ -64,7 +81,10 @@ argument. Finally, demonstrate calling `doSomething` with a function.**
 
 Your Answer:
 ```js
-// write code here
+function doSomething(thingToDo) {
+  return thingToDo;
+};
+function doSomething(makePizza) ;
```

Comment:
All you need to do here is call the function. Do you need to use the `function` keyword here to do that?